app/views/portal/show.rb

```diff
@@ -82,7 +82,6 @@ def content
           object: {
             annotations_later: true, # TODO: remove when styleguide assumes this
             creator: presenter.creator_title,
-            creators_info: presenter.creators_info,
```

Comment:
The new field group still needs to be constructed and passed to the template, i.e. with: `creators: presenter.field_group(:creators)`